### ISSUE TITLE
[TDS-TIA] Fix Knack Record ID PUT Request

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#7253]
Knack removed the ability to use API PUT Requests for Insert forms and they now only work with Edit forms. This field is critical for linking customer scopes & submissions to TIA cases.

Possible fix: Change the Knack Save Record ID page to an edit form

Alternative option: Use a different API call to grab the Knack Record ID

**Comment by ChrispinP** [comment]
Both my possible fix and alternative option above in the issue description were a troublesome no go if could be helped scenario.

the API View Based PUT Request is no longer needed as I have updated TDS with an updated workaround for linking customers to their TIA Portal since we can successfully use Page URL email field for the customer's side 
![image](https://user-images.githubusercontent.com/67915167/135175401-93fb0dd5-f8df-4196-93ee-8f1e2e47c370.png)


and use a Short Text field in the child objects to grab the Knack Record ID of the parent from the connection field and drop that in the Email link URL for the reviewer's side.

![image](https://user-images.githubusercontent.com/67915167/135175303-c598f30f-f355-4fda-b5b1-441ad0903b86.png)

I tested that emails are working with the changes and the records are connecting and grabbing the Knack Record ID correctly.

This is now complete, I suggested some cleanup tasks in issue #7084
